### PR TITLE
FCN 번역 연습

### PR DESCRIPTION
**Pull request by cepiloth**
## 라이선스 동의
_변경해주시는 내용에 BSD 3항 라이선스가 적용됨을 동의해주셔야 합니다._<br />
_더 자세한 내용은 [기여하기 문서](https://github.com/PyTorchKorea/hub-kr/blob/master/CONTRIBUTING.md)를 참고해주세요._<br />
_동의하시면 아래 `[ ]`를 `[x]`로 만들어주세요._<br />

- [ x] 기여하기 문서를 확인하였으며, 본 PR 내용에 BSD 3항 라이선스가 적용됨에 동의합니다.

## 관련 이슈 번호
_이 Pull Request와 관련있는 이슈 번호를 적어주세요._<br />
_이슈 또는 PR 번호 앞에 #을 붙이시면 제목을 바로 확인하실 수 있습니다. (예. #999 )_

- **이슈 번호**: #(숫자)

## PR 종류
_이 PR에 해당되는 종류 앞의 `[ ]`을 `[x]`로 변경해주세요._<br />
- [ ] 오탈자를 수정하거나 번역을 개선하는 기여
- [x ] 번역되지 않은 모델 소개를 번역하는 기여
- [ ] 공식 허브의 내용을 반영하는 기여
- [ ] 위 종류에 포함되지 않는 기여

## PR 설명
_이 PR로 무엇이 달라지는지 대략적으로 알려주세요._
파이토치 허브의 FCN 모델 소개 페이지가 한국어로 번역됩니다.

### CODE REVIEW

File: pytorch_vision_fcn_resnet101.md [hunk 1]
```diff
@@ -25,26 +25,23 @@ model = torch.hub.load('pytorch/vision:v0.10.0', 'fcn_resnet50', pretrained=True
 model.eval()
 ```
 
-All pre-trained models expect input images normalized in the same way,
-i.e. mini-batches of 3-channel RGB images of shape `(N, 3, H, W)`, where `N` is the number of images, `H` and `W` are expected to be at least `224` pixels.
-The images have to be loaded in to a range of `[0, 1]` and then normalized using `mean = [0.485, 0.456, 0.406]`
-and `std = [0.229, 0.224, 0.225]`.
+사전 학습된 모든 모델은 동일한 방식으로 정규화된 입력 이미지, 즉, H 와 W 는 최소 224 이상인 (3 x H x W) 형태의 3-채널 RGB 이미지의 미니 배치를 요구합니다. 이미지를 [0, 1] 범위에서 로드한 다음 mean = [0.485, 0.456, 0.406] 과 std = [0.229, 0.224, 0.225] 를 통해 정규화합니다.
```
Comment (zhoonit) [inline pytorch_vision_fcn_resnet101.md:28]: 미니 배치의 뜻을 살려 N 에 대한 설명도 추가되면 어떨까요?

**Review by cherimlee — COMMENTED** [review]
매끄러운 변역 감사드립니다. 백점입니다.

File: pytorch_vision_fcn_resnet101.md (hunk 1 above)
Comment (cepiloth) [reply pytorch_vision_fcn_resnet101.md:28]: 아하 미니배치 오늘 다시 공부하고서는 조금 반성했네요. 고려해보겠습니다.
